--- platform/linux-generic/arch/default/odp/api/abi/atomic_generic.h  2022-11-24 01:21:04.481324126 -0500
+++ platform/linux-generic/arch/default/odp/api/abi/atomic_generic.h  2022-11-24 01:20:59.687990595 -0500
@@ -81,7 +81,7 @@
 
 static inline odp_u128_t _odp_atomic_load_u128(odp_atomic_u128_t *atom)
 {
- odp_u128_t val;
+ odp_u128_t val = {0};
 
  *(__int128_t *)&val = __atomic_load_n((__int128_t *)&atom->v, __ATOMIC_RELAXED);
  return val;
